jest.config.js

```diff
@@ -0,0 +1,206 @@
+const path = require('path');
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+
+    // All imported modules in your tests should be mocked automatically
```

Comment:
the above comments on configuration property seems enough, we can omit all of these comments 🙏